### PR TITLE
Loc updates

### PR DESCRIPTION
**Pull request by cristianosuzuki77**
Latest translations

### CODE REVIEW

File: src/CodeStyle/Core/Analyzers/xlf/CodeStyleResources.zh-Hant.xlf [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
-  <file datatype="xml" source-language="en" target-language="zh-HANT" original="../CodeStyleResources.resx">
```
Comment (JoeRobich) [inline src/CodeStyle/Core/Analyzers/xlf/CodeStyleResources.zh-Hant.xlf:3]: @cristianosuzuki77 I notice many of these changes are just a change in casing of the target-language. Will the be consistent going forward?

**Review by JoeRobich — APPROVED** [review]
Just one question before this is good to merge.

File: src/CodeStyle/Core/Analyzers/xlf/CodeStyleResources.zh-Hant.xlf (hunk 1 above)
Comment (cristianosuzuki77) [reply src/CodeStyle/Core/Analyzers/xlf/CodeStyleResources.zh-Hant.xlf:3]: @JoeRobich yes, I'll ensure it.

Comment (JoeRobich) [reply src/CodeStyle/Core/Analyzers/xlf/CodeStyleResources.zh-Hant.xlf:3]: @cristianosuzuki77 Sounds good. Thanks!

**Pull request merged**